src/constants.py

```diff
@@ -78,6 +78,7 @@
 STATIC_CTOWN_HOURS_URL = DINING_STACK_URL + 'externalHours.json'
 STATIC_MENUS_URL = DINING_STACK_URL + 'hardcodedMenus.json'
 STATIC_EXPANDED_ITEMS_URL = DINING_STACK_URL + 'expandedItems.json'
+SWEETSPOT_ID = '8FpVJek1uqdaowPLZptfWg'
```

Comment:
add a comment here as to why this is necessary, just so it's clear for the future